cluster/src/dunit/scala/io/snappydata/cluster/ClusterManagerTestBase.scala

```diff
@@ -121,7 +121,8 @@ abstract class ClusterManagerTestBase(s: String)
       }
     }
 
-    Array(vm0, vm1, vm2).foreach(_.invoke(startNode))
+    vm0.invoke(startNode)
+    Array(vm1, vm2).map(_.invokeAsync(startNode)).foreach(_.getResult)
```

Comment:
@ymahajan starting all three asynchronously causes occasional lock timeouts on indexes/tables for the hive meta-store resulting in startup failure. Still need to look at the cause for that (since a separate dlock is acquired before hive meta-store initialization). For that reason I also had to do the same for snappy-start-all.sh script which now starts all servers in parallel except for the first one in a multi-node cluster (single localhost startup is still sequential as before). Will look into it when I get a chance.